content/en/metrics/explorer.md

```diff
@@ -11,6 +11,7 @@ further_reading:
   - link: "/metrics/distributions/"
```

Comment:
@estherk15  distributions shouldn't be linked as a subpage under metrics explorer . It should be moved back as its own page like summary or metrics without limits is. I'd prefer if it existed right under metric types in terms of ordering

Reply:
@ijkaylin I can do that! To clarify you want it at the same level under Metric Type? Or under as a child of Metric Type?

Reply:
https://datadoghq.atlassian.net/jira/software/projects/DOCS/boards/199

Reply:
Thanks for bringing that up @ijkaylin! 
I made a [request](https://datadoghq.atlassian.net/browse/DOCS-6447) for these changes on the board @estherk15 thank you for your help!